howto-promote.dd

```diff
@@ -94,4 +91,3 @@ $(H3 Promoting the Project)
 Macros:
 	TITLE=Promoting D Projects (or Internet Marketing 101)
 	WIKI=PromotingDProjects
```

Comment:
It's no longer used. Previously each page had a "page wiki" link, which lead to an associated page on the wiki, where users could add notes.